rstan/rstan/R/stan_plot.R

```diff
@@ -10,34 +10,34 @@ stan_trace <- function(object, pars, include = TRUE,
                        inc_warmup = FALSE,
                        nrow = NULL, ncol = NULL,
                        ..., window = NULL) {
-  
+
   .check_object(object, unconstrain)
   plot_data <- .make_plot_data(object, pars, include, inc_warmup, unconstrain)
-  
+
   thm <- rstanvis_theme()
   clrs <- rep_len(rstanvis_aes_ops("chain_colors"), plot_data$nchains)
-  
+
   base <-
     ggplot2::ggplot(
       plot_data$samp,
-      ggplot2::aes_string(x = "iteration", y = "value", color = "chain")
+      ggplot2::aes(x = .data$iteration, y = .data$value, color = .data$chain)
```

Comment:
I think there's no need for `.data$` with `aes()`. This applies everywhere.
```suggestion
      ggplot2::aes(x = iteration, y = value, color = chain)
```

Reply:
No, the `.data$` is necessary to avoid global variable name NOTEs in CRAN checks